src/pages/storybook.js

```diff
@@ -515,14 +515,6 @@ export default () => (
                     </GradientUnderline>
                 </H2>
             </Row>
-            <SectionHeader>Icons</SectionHeader>
```

Comment:
Was there a reason we wanted to remove these icons?

Reply:
There were duplicate `Icons` sections. I removed this extra section a few weeks ago but noticed its still here. Probably got re-added during a rebase.